src/traccuracy/track_errors/divisions.py

```diff
@@ -72,12 +75,15 @@ def _find_pred_node_matches(pred_node):
             ]
 
             # If daughters are same, division is correct
-            if Counter(succ_gt) == Counter(succ_pred):
+            cnt_gt = Counter(succ_gt)
+            cnt_pred = Counter(succ_pred)
+            if cnt_gt == cnt_pred:
                 g_gt.set_flag_on_node(gt_node, NodeFlag.TP_DIV, True)
                 g_pred.set_flag_on_node(pred_node, NodeFlag.TP_DIV, True)
-            # If daughters are at all mismatched, division is false negative
+            # If daughters are at all mismatched, division is a wrong child division
             else:
-                g_gt.set_flag_on_node(gt_node, NodeFlag.FN_DIV, True)
+                g_gt.set_flag_on_node(gt_node, NodeFlag.WC_DIV, True)
+                g_pred.set_flag_on_node(pred_node, NodeFlag.WC_DIV)
```

Comment:
How come we sometimes need to provide the boolean value of the flag, like in the line above, but other times we just provide the flag itself with no value?

Reply:
Valid question. The trues are left over from when we were being more explicit and densely annotating the graph. I'll clean it up so that's its consistent.